scripts/install/booklet.rdf

```diff
@@ -1,24 +1,52 @@
->
-  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet">
-    <rdfs:label xml:lang="en-US"><![CDATA[Booklet]]></rdfs:label>
-    <rdfs:comment xml:lang="en-US"><![CDATA[Booklet]]></rdfs:comment>
-    <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#TAOObject"/>
-  </rdf:Description>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xml:base="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        >
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet">
+        <rdfs:label xml:lang="en-US"><![CDATA[Booklet]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Booklet]]></rdfs:comment>
+        <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#TAOObject"/>
+    </rdf:Description>
     <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#BookletFile">
-    <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
-    <rdfs:label xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:label>
-    <rdfs:comment xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:comment>
-    <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
-    <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#File"/>
-    <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#AsyncFile"/>
-    <tao:TAOGUIOrder><![CDATA[20]]></tao:TAOGUIOrder>
-  </rdf:Description>
-  
- </rdf:RDF>
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Booklet File]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
+        <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#File"/>
+        <tao:TAOGUIOrder><![CDATA[20]]></tao:TAOGUIOrder>
+    </rdf:Description>
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Anonymous">
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Anonymous]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Indicate whether it should be anonymous]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
+        <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#RadioBox"/>
+        <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#Boolean"/>
+        <tao:TAOGUIOrder><![CDATA[30]]></tao:TAOGUIOrder>
+    </rdf:Description>
+
```

Comment:
The both properties are not correct. They should be :

This should be replaced by:

```xml
<rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Test">
     <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
     <rdfs:label xml:lang="en-US"><![CDATA[Test]]></rdfs:label>
     <rdfs:comment xml:lang="en-US"><![CDATA[Booklet's Test]]></rdfs:comment>
     <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/TAOTest.rdf#Test"/>
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
     <generis:Multiple rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#True" />
     <tao:TAOGUIOrder><![CDATA[32]]></tao:TAOGUIOrder>
</rdf:Description>

<rdf:Description rdf:about="http://www.tao.lu/Ontologies/Booklet.rdf#Groups">
     <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
     <rdfs:label xml:lang="en-US"><![CDATA[Groups]]></rdfs:label>
     <rdfs:comment xml:lang="en-US"><![CDATA[Group of the booklets]]></rdfs:comment>
     <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/TAOGroup.rdf#Group"/>
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/Booklet.rdf#Booklet"/>
     <generis:Multiple rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#True" />
     <tao:TAOGUIOrder><![CDATA[40]]></tao:TAOGUIOrder>
</rdf:Description>
```